stm32cube/stm32g0xx/README

```diff
@@ -23,7 +23,7 @@ URL:
    https://github.com/STMicroelectronics/STM32CubeG0
 
 Commit:
-   f4e8c0a878aebc1b873b8590fd29563404759694
+   a019f83e7693348cf50f1466756ea985d3a96163
 
```

Comment:
Please fix Patch list:

Looks like "*Wrap define UNUSED with an ifndef" has been fixed

Reply:
I removed the "Wrap define UNUSED with an ifndef" from the README  patch list.

Reply:
Done